### PR TITLE
chore(docs): add doc about metapartition

### PR DESCRIPTION
**Pull request by akhilerm**
- include details about meta partition in device localpv

Signed-off-by: Akhil Mohan <akhil.mohan@mayadata.io>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,14 @@ truncate -s 1024G /tmp/disk.img
 sudo losetup -f /tmp/disk.img --show
 ```
 
+###### MetaPartition
+A meta partition is a small (~10MiB) partition that needs to be created on the disk, so as store the disk identification
+information. In device localpv, the partition name of the meta partition is used to identify the disk. A partition
+should follow the below criteria, so that openebs identifies it as a meta partition and use it for disk identification
+1. The partition number should be 1 (`ID_PART_ENTRY_NUMBER=1`)
+2. The partition should not be formatted with any filesystem
```
Comment (pawanpraka1) [inline README.md:49]: do we verify that there is no filesystem or partitions does not have any flags set.

Comment (akhilerm) [reply README.md:49]: Nope. We are not checking for filesystem and flags, only partition number is checked. cc: @praveengt Do we need to add this check?

Comment (praveengt) [reply README.md:49]: https://github.com/openebs/device-localpv/blob/ca58e5841c7a1a3ed74cb8e0913a98a6e678ad33/pkg/device/device-util.go#L411
Checking the column number to 5, will work only if flags and FS type did not exist

**Review by praveengt — APPROVED** [review]

**Review by pawanpraka1 — APPROVED** [review]
looks good.

File: README.md (hunk 1 above)
Comment (akhilerm) [reply README.md:49]: Thank you. Didnt know about that.

**Pull request merged**